app1.py

```diff
@@ -9,22 +9,40 @@
 from streamlit_folium import folium_static
 from folium import plugins
 from country_statistics import show_country_statistics
+#import Image
 
 def app():
 
-    st.markdown(
-        """
-        <h1 style='text-align: center; color: black;'><u>Globalizer</u></h1>
+    title_container = st.container()
```

Comment:
Maybe remove that image. It disturbs the page because it reloads every time a setting is changed.